aml/org.testeditor.aml.dsl.tests/src/org/testeditor/aml/dsl/tests/common/AmlTestModels.xtend

```diff
@@ -28,18 +30,41 @@ class AmlTestModels {
 			interactionTypes += startInteraction
 			
 			val interactionTypeForGetSome = interactionType("getValue") => [
-				defaultMethod = methodReference(resourceSet, DummyFixture, "getValue").withLocatorStrategy
-				template = template("getValue")
+				defaultMethod = methodReference(resourceSet, DummyFixture, "getValue", "element")
+				template = template("getValue").withParameter(defaultMethod.parameters.head)
 			]
-			interactionTypes += interactionTypeForGetSome		
+			interactionTypes += interactionTypeForGetSome
+			
+			val interactionTypeForWait = interactionType("wait") => [
```

Comment:
Can we get rid of this and use `org.testeditor.dsl.common.testing.DummyFixture.getAmlModel()` instead?